models/article-model.js

```diff
@@ -0,0 +1,62 @@
+const db = require("../db/connection.js");
+
+
+const selectArticles = () => {
+    return db.query(`SELECT * FROM articles`);
+  };
+  
+  const selectArticleById = (article_id) => {
+  
+      return db
+      .query(`SELECT articles.*, COUNT(articles.article_id) AS comment_count FROM articles JOIN comments ON comments.article_id = articles.article_id WHERE articles.article_id=$1 GROUP BY articles.article_id`, [article_id])
+      .then(({ rows: article }) => {
+       
+        if (article.length > 0) {
+          return article[0];
```

Comment:
In this case, "article" doesn't reflect the article, it reflect the rows array that article is being returned within. Therefore article[0] isn't readable - what is 0 of article? This should either be rows[0] or article should be destructured (currently just being renamed from rows) on  line 12